Toolset/palettes/script editor/behaviors/revsestackbehavior.livecodescript

```diff
@@ -41,6 +41,12 @@ local sLastShownExecutionPoint
 #   Sets the current object to edit to pObject. If pStartingLine is given, then seeks to that line
 #   otherwise seeks to the last selected line.
 command revSESetCurrentObject pObject, pStartingLine, pDontSendCallbacks
+   lock screen
+   
+   revSEGetCurrentObject
+   if the result is not empty and the long id of pObject is the long id of the result then
```

Comment:
That's a good plan. @peter-b could we add that to the backlog? Maybe it could be part of the object handle variable type we discussed? FYI @livecodeali it was a value with an underlying pre-resolved object reference. When used as a string it would have a long id representation but otherwise need no parsing as it was used.